features/git-extract/cherry_pick_conflict/with_remote_origin/with_open_changes.feature

```diff
@@ -34,7 +33,7 @@ Feature: git extract: resolving conflicts between main branch and extracted comm
       To continue after you have resolved the conflicts, run "git extract --continue".
       """
     And I end up on the "refactor" branch
-    And I don't have my uncommitted file
+    And my uncommitted file is still stashed away
```

Comment:
Not a big fan of the wording, "still" to me implies it was stashed away before and thats not the case.
I'd prefer "And my uncommitted file in stashed".